packages/clay-badge/src/__tests__/__snapshots__/ClayBadge.tsx.snap

```diff
@@ -0,0 +1 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I would have expected there to be some content in here.

Reply:
Updated this in the generator